packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -166,6 +167,10 @@ export const authenticatedStatusMiddleware = async (
   next: () => Promise<unknown>
 ): Promise<void> => {
   ctx.authenticated = false
+  if (!ctx.request.headers.authorization) {
```

Comment:
now that we expect the request to be for sure authorized after this check, we can get rid of the try-catch below altogether